universum.cabal

```diff
@@ -107,7 +107,7 @@ library
                      , stm
                      -- Make sure that "toString-toText-rewritting" note
                      -- is still valid when bumping this constraint.
-                     , text >= 1.0.0.0 && <= 2.0.2
+                     , text >= 1.0.0.0 && <= 2.1.1
```

Comment:
I checked this change:
1. There seems to be no changes to `pack`/`unpack` functions or their rewrite rules in `text` package. Please update the commit mentioned in `toString-toText-rewritting` note in Haskell code to `c826a6cb3d29242ce3632b490670ae7997689eaa` (current `origin/master` of text package)
2. Benchmarks produce the expected result.

Reply:
Added "in place" (in the old commit).